functests/utils/profiles/profiles.go

```diff
@@ -32,3 +35,15 @@ func GetByNodeLabels(nodeLabels map[string]string) (*performancev1alpha1.Perform
 
 	return result, nil
 }
+
+// GetProfileMessage gets the performance profile message for the given type
+func GetProfileMessage(nodeLabels map[string]string, conditionType v1.ConditionType) string {
```

Comment:
This name is misleading. The func returns a condition reason. It's ok to skip profile in the name, we are in the profile package. So I'd suggest GetConditionReason()

Reply:
Done